galsim/wfirst/wfirst_bandpass.py

```diff
@@ -44,10 +44,22 @@ def getBandpasses(AB_zeropoint=True, exptime=None, default_thin_trunc=True, **kw
     because the stored 'official' versions of the bandpasses cover a wide wavelength range.  So even
     if thinning is not desired, truncation is recommended.
 
-    By default, the routine will set an AB zeropoint using the WFIRST effective diameter and default
-    exposure time.  Setting the zeropoint can be avoided by setting `AB_zeropoint=False`; changing
-    the exposure time that is used for the zeropoint calculation can be used by setting the
-    `exptime` keyword.
+    By default, the routine will set an AB zeropoint (unless `AB_zeropoint=False`).  The
+    zeropoint in GalSim is defined such that the flux is 1 photon/cm^2/sec through the
+    bandpass. This differs from an instrumental bandpass, which is typically defined such that the
+    flux is 1 photon/sec for that instrument.  The difference between the two can be calculated as
+    follows:
+
+            # Calculate effective diameter given the obscuration, then get the collecting area in
+            # cm^2.
+            d_eff = 100. * galsim.wfirst.diameter * np.sqrt(1.-galsim.wfirst.obscuration**2)
+            area_eff = np.pi * (d_eff/2.)**2
```

Comment:
There is already a `galsim.wfirst.collecting_area` variable, which I think is this value.  Should maybe recommend using that instead of this calculation.

Reply:
Yes, that's absolutely right.  Fixed.